web/src/components/featuresSection.js

```diff
@@ -7,7 +7,7 @@ import tw from 'twin.macro';
 import Feature from './feature';
 
 const content = {
-    heading: 'Why Hubble?',
+    heading: '',
     features: [
         {
             heading: 'Compliance and Security',
```

Comment:
are we still using these (should be in sanity)? If not let's remove them.

Reply:
@asasmith these are old components I referred to from oldIndex. They arent being used other than how they are being stored via OldIndex.js

Reply:
Same comment as above. If we're not suing these components then why is this is the diff?

Reply:
@asasmith because I had removed them previously, I think in another PR. How would you like me to proceed?

Reply:
@asasmith that can be removed

Reply:
@asasmith actually it cant be removed, or itll throw an error. I removed the content, but kept the variable declaration

Reply:
Ok, can you create an issue to remove everything related to the original landing page (`oldIndex`) so that we remember to to this clean up?